table/sstable/builder.go

```diff
@@ -202,7 +202,7 @@ func (b *Builder) Reset(f *os.File) {
 	b.resetBuffers()
 	b.w.Reset(f)
 
-	modFile, err := y.OpenTruncFile(ModelFilename(b.file.Name(), b.level), false)
+	modFile, err := y.OpenTruncFile(ModelFilename(b.file.Name()), false)
```

Comment:
I don't think those `levels` are needed as I don't see them on other files? 🤔

Reply:
yes, I mark a level on filename so we can know what level it is  more convenient